packages/story-editor/src/components/panels/design/link/link.js

```diff
@@ -282,6 +292,9 @@ function LinkPanel({ selectedElements, pushUpdateForObject }) {
               </IconText>
             </IconInfo>
           </Row>
+          {!isMultipleRel && (
+            <LinkRelations onChangeRel={onChangeRel} rel={link?.rel || []} />
```

Comment:
Same here, optional chaining is superfluous, `link` doesn't seem to be nullable.